Github CI dosen't work

Check Github Action Script due to errors with code coverage step

It is seems to be the `python-dotenv` module which is missing.  
```
___________________ ERROR collecting tests/settings_test.py ____________________
ImportError while importing test module '/home/runner/work/blask/blask/tests/settings_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/settings_test.py:6: in <module>
    from dotenv import load_dotenv
E   ModuleNotFoundError: No module named 'dotenv'
```

- I can add it on `Pipfile` or just install it during the ci, but i think if we want to run test manually it's better to have it on `Pipfile`.

- I have a suggestion for the ci, i think it's more relevant if the CI run when a pull request is submitted instead of when commits are pushed on branches.

What do you think @zerasul about this points ?

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -1,12 +1,12 @@
 name: blask Build
 on:
-  push:
+  pull_request:
     branches:
       - master
       - githubactions
       - develop
 
-env: 
+env:
   SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   ACTIONS_ALLOW_UNSECURE_COMMANDS: true
@@ -32,6 +32,7 @@ jobs:
         run : |
             pip install pipenv
             pipenv install importlib-metadata
+            pipenv install python-dotenv
             pipenv install --dev
 
       - name: Code coverage

